src/clj_jargon/init.clj

```diff
@@ -192,10 +202,11 @@
   (let [[opts-map [[cm-sym] & body]] (split-with #(not (vector? %)) params)
          opts                        (apply hash-map opts-map)]
     `(let [auto-close#  (if (nil? (:auto-close ~opts)) true (:auto-close ~opts))
-           client-user# (if (:client-user ~opts) (:client-user ~opts) (:username ~cfg))]
+           client-user# (if (:client-user ~opts) (:client-user ~opts) (:username ~cfg))
+           lazy# (if (nil? (:lazy ~opts)) false (:lazy ~opts))]
```

Comment:
Could this entire `(if ...)` be replaced with `(:lazy ~opts false)`?

Reply:
Heh, quite possibly. I was copying the `auto-close#` above is why it's this way, though.